packages/world/ts/library/config/worldConfig.ts

```diff
@@ -9,7 +9,7 @@ const zSystemAccessList = z.array(zSystemName.or(zEthereumAddress)).default(SYST
 // The system config is a combination of a name config and access config
 const zSystemConfig = z.intersection(
   z.object({
-    name: zSelector.optional(),
+    name: zName.optional(),
```

Comment:
ahh I guess if we adjusted `zName` to validate length, it'd fail for system contract names?